Class1.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
```

Comment:
This file needs to be deleted... It looks like a duplicate of DeveloperGameClass

You deleted it from your solution, but that doesn't remove it from the folder in your file system.

Try to carefully look over each file you are going to check in before checking it in.

Reply:
OK, I see. I'm working on it right now!

Reply:
I fixed this issue.

Reply:
@paulaOliverosHollis There is only 1 commit on this pull request, so I'm not sure if you forgot to push or if you checked it into the wrong place. Please update this on your "CreatingDeveloperGameClass"

Reply:
![image](https://user-images.githubusercontent.com/43797367/62722908-70081680-b9dd-11e9-853a-a058a3bdc2dc.png)

Reply:
Please check now.

Reply:
Yep, I see it now.

Reply:
Please check the pull request to make sure it is updated before saying it's fixed. :)